### PR TITLE
enable fedora 30 testing and disable 28, py3 only for rhel8 & fedora

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: osbs-client.spec [hunk 1]
```diff
@@ -155,12 +156,13 @@ This package contains osbs Python 3 bindings.
 %py3_install
 mv %{buildroot}%{_bindir}/osbs %{buildroot}%{_bindir}/osbs-%{python3_version}
 ln -s  %{_bindir}/osbs-%{python3_version} %{buildroot}%{_bindir}/osbs-3
-%endif # with_python3
-
+ln -s  %{_bindir}/osbs-%{binaries_py_version} %{buildroot}%{_bindir}/osbs
```
Comment (twaugh) [inline osbs-client.spec]: Looks like this line (along with line 164) could be moved outside of the %if.

File: osbs-client.spec [hunk 2]
```diff
@@ -175,27 +177,11 @@ LANG=en_US.utf8 py.test-%{python2_version} -vv tests
...
-%ghost %config(noreplace) %{_datadir}/osbs/orchestrator_customize.json
-%ghost %config(noreplace) %{_datadir}/osbs/prod_customize.json
-%ghost %config(noreplace) %{_datadir}/osbs/worker_customize.json
-
 
 %if 0%{?with_python3}
 %files -n python3-osbs-client
-%doc README.md
-%{!?_licensedir:%global license %doc}
-%license LICENSE
```
Comment (athos-ribeiro) [inline osbs-client.spec]: Shouldn't all the subpackages ship the LICENSE file?

File: osbs-client.spec (hunk 1 above)
Comment (rcerven) [reply osbs-client.spec]: ok

File: osbs-client.spec (hunk 2 above)
Comment (rcerven) [reply osbs-client.spec]: yeah will change it

**Comment by rcerven** [comment]
rn updated: enable building py3 package for rhel8

**Pull request merged**